infra/aws/e2e/accounts/tf/modules/account-and-user/main.tf

```diff
@@ -0,0 +1,40 @@
+# Intermediate module; workaround for terraform being bad at multiple providers
```

Comment:
I'm not sure I understand what problem this is solving.  Why not merge the account and user modules into this one module?